packages/amplify-provider-awscloudformation/src/initializer.ts

```diff
@@ -30,7 +30,7 @@ export async function run(context) {
     context.exeInfo = context.exeInfo || {};
     const { projectName } = context.exeInfo.projectConfig;
     const initTemplateFilePath = path.join(__dirname, '..', 'resources', 'rootStackTemplate.json');
-    const timeStamp = `${moment().format('Hmmss')}`;
+    const timeStamp = `${moment().format(process.env.CIRCLECI ? 'HmmssSSS' : 'Hmmss')}`;
```

Comment:
would a truncated UUID be even better / simpler?

Reply:
yes, will change